habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -379,6 +379,15 @@ def _eval_checkpoint(
         self.agent.load_state_dict(ckpt_dict["state_dict"])
         self.actor_critic = self.agent.actor_critic
 
+        # get name of performance metric, e.g. "spl"
+        metric_name = self.config.TASK_CONFIG.TASK.MEASUREMENTS[0]
```

Comment:
Let's add a comment to default config, that first measure is main measure for Trainers.